src/viam/app/data_client.py

```diff
@@ -371,13 +371,21 @@ async def get_latest_tabular_data(
 
         ::
 
-            time_captured, time_synced, payload = await data_client.get_latest_tabular_data(
-                part_id="<PART-ID>",
-                resource_name="<RESOURCE-NAME>",
-                resource_subtype="<RESOURCE-SUBTYPE>",
-                method_name="<METHOD-NAME>"
+            tabular_data = await data_client.get_latest_tabular_data(
+                part_id="77ae3145-7b91-123a-a234-e567cdca8910",
```

Comment:
(q) is this a real part-id? Is there any reason we might want to redact it?

Reply:
Not a real part-id!